api/config/cors.php

```diff
@@ -15,7 +15,7 @@
     |
     */
 
-    'paths' => ['api/*', 'sanctum/csrf-cookie'],
+    'paths' => ['*'],
```

Comment:
Oops, CORS has been broken on our API.  Do we want to restrict it or is a wildcard OK?